connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConnectorTest.java

```diff
@@ -121,11 +131,15 @@ public void testStartupAndShutdown() {
 
         replayAll();
 
-        WorkerConnector workerConnector = new WorkerConnector(CONNECTOR, connector, ctx, listener);
+        WorkerConnector workerConnector = new WorkerConnector(CONNECTOR, connector, ctx, metrics, listener);
 
+        assertStoppedMetric(workerConnector);
```

Comment:
A different way of interpreting the enum `INIT` is `Initializing` rather than `Initialized`. Maybe that phrasing would make these tests clearer? Using `stopped` here seems confusing because it seems weird to go from stopped -> initialized.